rocketpool-cli/service/config/addons.go

```diff
@@ -39,9 +40,17 @@ func NewAddonsPage(home *settingsHome) *AddonsPage {
 
 	// Create the addon subpages
 	addonsPage.gwwPage = NewAddonGwwPage(addonsPage, home.md.Config.GraffitiWallWriter)
+
 	addonSubpages := []settingsPage{
 		addonsPage.gwwPage,
 	}
+
+	// TODO: Make this respond to uncommitted config changes
```

Comment:
Currently the addon only appears/disappears once you have save your changes to the "MetricsEnabled" setting. Is there a way to fetch the current uncommitted state of the config?